deploy/olm-catalog/performance-addon-operator/0.0.1/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -31,6 +31,11 @@ spec:
         spec:
           description: PerformanceProfileSpec defines the desired state of PerformanceProfile.
           properties:
+            additionalKernelArgs:
+              description: Addional kernel arguments.
+              items:
+                type: string
+              type: array
```

Comment:
the golang json encoder will reorder map keys. But this is an array, so it should be fine.